### DISCUSSION TITLE
Database driver change

### DISCUSSION THREAD

**Discussion by ffrhtt** [Laravel-Backpack/community-forum#302] [General] [answered]
Hi, I want to change my database, I am using sqlsrv (MS SQL) now. I try change to Mysql driver in env file but backpack not changing. It still use sqlsrv. for detail 

https://stackoverflow.com/q/65447505/3129822

**Comment by tabacitu** [comment, marked as answer]
Heads-up @ffrhtt - Backpack will use whatever the Eloquent Model uses for the DB Driver. So this isn't Backpack-specific - it's more of a Laravel question.
